x/gov/keeper/proposal_test.go

```diff
@@ -296,7 +297,9 @@ func (suite *KeeperTestSuite) TestCancelProposal() {
 
 func TestMigrateProposalMessages(t *testing.T) {
```

Comment:
Don't need this test

Reply:
Yeah right, but we are trying to minimize the diffs with upstream for future pulls, so we're leaving also tests which we don't need, unless they fail (in which case we fix according to `heimdall` logic, or explicitly `skip`)